tests/trestle/core/commands/assemble_test.py

```diff
@@ -0,0 +1,67 @@
+# -*- mode:python; coding:utf-8 -*-
+
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for trestle assemble command."""
+import pathlib
+import shutil
+import sys
+from unittest import mock
+
+from trestle.cli import Trestle
+from trestle.oscal.catalog import Catalog
+from trestle.utils.load_distributed import load_distributed
```

Comment:
So food for thought - I wonder whether with some of these modules we should expose interior functions up to the parent module level. 

e.g. `trestle.utils.load_distributed` lets discuss at some point (no need to change here.)

Reply:
I think it'll feel very satisfying to import modules that way!